app/experimenter/experiments/tests/factories.py

```diff
@@ -178,6 +208,13 @@ class Lifecycles(Enum):
     LAUNCH_APPROVE_TIMEOUT = LAUNCH_APPROVE_WAITING + (LifecycleStates.DRAFT_REVIEW,)
     LIVE_ENROLLING = LAUNCH_APPROVE_APPROVE + (LifecycleStates.LIVE_IDLE_ENROLLING,)
     LIVE_PAUSED = LIVE_ENROLLING + (LifecycleStates.LIVE_IDLE_PAUSED,)
+
+    LIVE_IDLE = (LifecycleStates.LIVE_IDLE,)  # why didn't we have this one before?
+    LIVE_DIRTY = (LifecycleStates.LIVE_DIRTY,)
+    LIVE_REVIEW_REQUESTED = (LifecycleStates.LIVE_REVIEW,)
+    LIVE_REVIEW_APPROVED = (LifecycleStates.LIVE_APPROVED,)
+    LIVE_REVIEW_WAITING = (LifecycleStates.LIVE_WAITING,)
```

Comment:
Oh this isn't exactly how the lifecycles work.  This would create an experiment that goes straight from nothing straight to each of these states without passing through any of the states before it.  Like it just popped into existence like a virtual quantum particle.  That's fine for the quantum vacuum of outer space, but Experimenter is not so stochastic.  Look above at how they each take one of the previous states and then tack on another, that's what we'd want here.

Also a plain `LIVE_IDLE` is the `LIVE_ENROLLING` one above, since there's two cases where an experiment would be `LIVE_IDLE`, with `is_paused=False` and `is_paused=True`.  Also just looking at it here I realize the `LIVE_PAUSED` one should have these `LIVE_REVIEW_....` ones in between `LIVE_ENROLLING` and `LIVE_PAUSED` since you have to go through the full review flow (minus DIRTY) to pause it.  Wouldn't hurt to update that while you're adding the other review states for live.

Reply:
👩‍🚀 that makes so much more sense. I was a little confused with the additions here and why they were defined like that